spec/unit/product_spec.rb

```diff
@@ -0,0 +1,79 @@
+require './lib/modules/product.rb'
+
+RSpec.describe Store::Product do 
+  let(:name) { 'foo' }
+  let(:price) { 1 }
+  let(:vat) { 23 }
+
+  subject(:product) { Store::Product.new(name, price, vat) }
+
+  describe :new do
+    context :params do
```

Comment:
context "for params"

or

context "for invalid params" - and in this case you can remove the first spec